main.tf

```diff
@@ -1,30 +1,38 @@
 resource "aws_lambda_permission" "resource" {
   function_name = "${var.function_name}"
-  statement_id = "AllowExecutionFromApiGateway"
-  action = "lambda:InvokeFunction"
-  principal = "apigateway.amazonaws.com"
+  statement_id  = "AllowExecutionFromApiGateway"
+  action        = "lambda:InvokeFunction"
+  principal     = "apigateway.amazonaws.com"
 }
 
 resource "aws_api_gateway_method" "method" {
-  rest_api_id = "${var.api_id}"
-  resource_id = "${var.resource_id}"
-  http_method = "${var.http_method}"
+  rest_api_id   = "${var.api_id}"
+  resource_id   = "${var.resource_id}"
+  http_method   = "${var.http_method}"
   authorization = "NONE"
 }
 
 resource "aws_api_gateway_integration" "integration" {
-  rest_api_id = "${var.api_id}"
-  resource_id = "${var.resource_id}"
-  http_method = "${aws_api_gateway_method.method.http_method}"
+  rest_api_id             = "${var.api_id}"
+  resource_id             = "${var.resource_id}"
+  http_method             = "${aws_api_gateway_method.method.http_method}"
   integration_http_method = "${aws_api_gateway_method.method.http_method}"
-  type = "AWS_PROXY"
-  uri = "${var.invoke_arn}"
+  type                    = "AWS_PROXY"
+  uri                     = "${var.invoke_arn}"
 }
 
 # CORS
 
 module "CORS" {
-  source = "github.com/carrot/terraform-api-gateway-cors-module"
+  source      = "github.com/carrot/terraform-api-gateway-cors-module"
   resource_id = "${var.resource_id}"
   rest_api_id = "${var.api_id}"
 }
+
+output "lambda_invoke_arn" {
+  value = "${aws_api_gateway_integration.integration.uri}"
+}
+
+output "http_method" {
```

Comment:
Wouldn't the `lambda_invoke_arn` be enough for this solution? 

Especially since `http_method` is already fed in by the user.

Reply:
It probably would be.  The difference being this <code>http_method</code> output is tied to the created resource's <code>http_method</code>, rather than the variable.  This would allow independent validation that both resources are created.  I'm sure it's not required, however.